models/Settings.php

```diff
@@ -16,13 +16,18 @@ class Settings extends Model
     public $settingsCode = 'user_settings';
     public $settingsFields = 'fields.yaml';
 
+
+    const PASSWORD_LENGTH_KEY = 'password_length';
```

Comment:
Why is this required?

Reply:
It prevents developers from using String keys and helps refactoring the code later. But we can cut it off.